bowtie.wdl

```diff
@@ -0,0 +1,43 @@
+task Bowtie_se {
+    File reference
+    File read1
+    String sam
+    String? precommand
+    Int? threads
+    String? otherOptions
+
+    command {
+        ## https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+        set -eo pipefail
+        mkdir -p
```

Comment:
Which dir are you creating here?
I always use `mkdir -p $(dirname ${sam})` to make sure any file creation attempts with the `>` sign work.